core/domain/recommendations_services.py

```diff
@@ -160,7 +160,7 @@ def get_topic_similarity(topic_1, topic_2):
     the topics are the same.
     """
 
-    if (topic_1 in RECOMMENDATION_CATEGORIES and
+    if (topic_1 in RECOMMENDATION_CATEGORIES) and (
```

Comment:
I think it would be better if we go by the previous brace style for logical operators. This need not be changed.

Reply:
Also, please correct this at other places too.